azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/VideoViewManager.kt

```diff
@@ -6,26 +6,36 @@ package com.azure.android.communication.ui.presentation
 import android.content.Context
 import android.view.View
 import android.view.ViewGroup
-import com.azure.android.communication.calling.CreateViewOptions
-import com.azure.android.communication.calling.MediaStreamType
 import com.azure.android.communication.calling.RemoteVideoStream
-import com.azure.android.communication.calling.ScalingMode
 import com.azure.android.communication.calling.VideoStreamRenderer
 import com.azure.android.communication.calling.VideoStreamRendererView
+import com.azure.android.communication.calling.MediaStreamType
+import com.azure.android.communication.calling.ScalingMode
+import com.azure.android.communication.calling.CreateViewOptions
 import com.azure.android.communication.ui.service.calling.sdk.CallingSDKWrapper
 
 internal class VideoViewManager(
     private val callingSDKWrapper: CallingSDKWrapper,
     private val context: Context,
 ) {
-    class VideoRenderer constructor(
+    private val remoteParticipantVideoRendererMap: HashMap<String, VideoRenderer> = HashMap()
+    private val localParticipantVideoRendererMap: HashMap<String, VideoRenderer> = HashMap()
+
+    private class VideoRenderer(
         var rendererView: VideoStreamRendererView?,
         var videoStreamRenderer: VideoStreamRenderer?,
         var videoStreamID: String,
+        var isScreenShareView: Boolean
     )
 
-    private val remoteParticipantVideoRendererMap: HashMap<String, VideoRenderer> = HashMap()
-    private val localParticipantVideoRendererMap: HashMap<String, VideoRenderer> = HashMap()
+    fun getScreenShareVideoStreamRenderer(): VideoStreamRenderer? {
```

Comment:
This can be done a bit more functionally.

    fun getScreenShareVideoStreamRenderer() =
        remoteParticipantVideoRendererMap.values.find { it.isScreenShareView }?.videoStreamRenderer